noxfile.py

```diff
@@ -43,35 +63,38 @@ def _should_install(session: nox.Session) -> bool:
     return should_install
 
 
-@nox.session(reuse_venv=True)
+@session()
 def lint(session: nox.Session) -> None:
     """Check the themes pre-commit before any other session."""
     session.install("pre-commit")
     session.run("pre-commit", "run", "-a")
 
 
-@nox.session()
+@session()
 def compile(session: nox.Session) -> None:
-    """Compile the theme's web assets with sphinx-theme-builder."""
+    """Compile the theme's web assets."""
     if _should_install(session):
         session.install("-e", ".")
         session.install("sphinx-theme-builder[cli]")
+        session.install("Babel")
 
     session.run("stb", "compile")
+    session.run("pybabel", "compile", "-d", str(locale_dir), "-D", "sphinx")
 
 
-@nox.session()
+@session()
 def docs(session: nox.Session) -> None:
     """Build the documentation and place in docs/_build/html. Use --no-compile to skip compilation."""
     if _should_install(session):
```

Comment:
were you intending to get rid of this? (I thought so but maybe I misunderstood)

Reply:
nope I just wanted t o use my custom session instead. I even plan to check the warnings in the static build as well in a future nox/CI PR